lute/std/libs/system/init.luau

```diff
@@ -7,26 +7,27 @@ local system = require("@lute/system")
 local path = require("@std/path")
 local platforminfo = require("@self/platforminfo")
```

Comment:
probably this should just be called `platform`? `platforminfo` feels like A Lot to me